EgammaAnalysis/ElectronTools/plugins/CalibratedElectronProducersRun2.cc

```diff
@@ -55,7 +55,7 @@ void
 CalibratedElectronProducerRun2T<T>::produce( edm::Event & iEvent, const edm::EventSetup & iSetup ) 
 {
     iSetup.get<GBRWrapperRcd>().get(theGBRForestName, theGBRForestHandle);
-    theEpCombinationTool.init(theGBRForestHandle.product());
+    theEpCombinationTool.init(theGBRForestHandle.product() );
```

Comment:
@emanueledimarco 
Usually we should avoid this kind of changes unless it is really necessary.